lib/rubocop/cop/style/file_null.rb

```diff
@@ -42,10 +47,22 @@ class FileNull < Base
         REGEXP = %r{\A(/dev/null|NUL:?)\z}i.freeze
         MSG = 'Use `File::NULL` instead of `%<source>s`.'
 
+        def on_new_investigation
+          return unless (ast = processed_source.ast)
+
+          @contain_dev_null_string_in_file = ast.each_descendant(:str).any? do |str|
+            content = str.str_content
+
+            valid_string?(content) && content.downcase == '/dev/null' # rubocop:disable Style/FileNull
+          end
```

Comment:
I think this might be too broad. For example, this would register as an offense since it contains a `str` node with `/dev/null`:

```ruby
"#{foo}/dev/null"
```

Reply:
Actually, if the term `/dev/null` is used, it is likely related to the filesystem. Therefore, unless there is a specific issue, I think it can be better to keep the current simple implementation as is.